src/components/post-details-page/PostDetailsPage.js

```diff
@@ -1,34 +1,52 @@
 import React, { Component } from 'react';
+import {accessToken} from "../../constants";
+import {PostCard} from "../post-card/PostCard";
 
 class PostDetailsPage extends Component {
 
   constructor(props) {
     super(props);
 
     this.state = {
-      post: null
+      post: null,
       // todo 3: добавить isLoading флажок - индикатор загрузки
+      isLoading: false
     };
   }
 
   componentDidMount() {
     // todo 3: вызвать загрузку инфрмации про пост loadPost
+    this. loadPost();
   }
 
   loadPost = async () => {
     // todo 3:
     //  достать id поста из props посредством пропсов, которые дает нам роутер
+    const {match: {params: {Id}}} =  this.props;
     //  проверьте лежит ли ваш accessToken в constants/index.js
     //  прежде чем отправить запрос - включите в true флажок загрузки в стейте isLoading
+    await this.setState({
```

Comment:
You don't need to use await with this.setState
It's async function but it runs very fast  and no need to block the thread till it would be finished